tt_metal/jit_build/build.cpp

```diff
@@ -464,6 +464,9 @@ void JitBuildState::link(const string& log_file, const string& out_dir) const {
     cmd = "cd " + out_dir + " && ";
     cmd += env_.gpp_;
     cmd += lflags;
+    if (this->target_name_ == "idle_erisc") {
```

Comment:
this part make sense in general but isn't needed in this instance, right?  the linker scripts are processed w/ the offline build and so the ifdefs are already expanded?

Reply:
ah okay so I guess I should be using the `COMPILE_FOR_IDLE_ERISC` env var